src/components/compound/DlTable/DlTable.vue

```diff
@@ -317,102 +351,106 @@
                         :cols="computedCols"
                     />
                     <slot
-                        v-for="(row, pageIndex) in computedRows"
-                        v-bind="
-                            getBodyScope({
-                                key: getRowKey(row),
-                                row,
-                                pageIndex,
-                                trClass: isRowSelected(getRowKey(row))
-                                    ? 'selected'
-                                    : ''
-                            })
-                        "
                         name="body"
+                        :computed-rows="computedRows"
```

Comment:
This is a huge breaking change for anyone who used the API beforehand 
how does this work ?